### PR TITLE
chore(multi-env): show sub and resource group in env tree

### PR DESCRIPTION
**Pull request by dooriya**
If the env has already been provisioned, then the resource group and sub info will be showed in the tree view:
image

### CODE REVIEW

File: packages/vscode-extension/src/utils/commonUtils.ts [hunk 1]
```diff
@@ -211,3 +217,80 @@ export function getAllFeatureFlags(): string[] | undefined {
 export function getIsExistingUser(): string | undefined {
   return ext.context.globalState.get<string>(UserState.IsExisting);
 }
+
+export function getDefaultEnv(): string {
+  return isMultiEnvEnabled() ? "dev" : "default";
+}
```
Comment (qinezh) [inline packages/vscode-extension/src/utils/commonUtils.ts:223]: As we already have `environmentManager.getDefaultEnvName()`, there's no need to create a duplicated one?

Comment (dooriya) [reply packages/vscode-extension/src/utils/commonUtils.ts:223]: It makes sense. Let me fix it

File: packages/vscode-extension/src/envTree.ts [hunk 2]
```diff
@@ -117,3 +119,43 @@ async function localSettingsExists(projectRoot: string): Promise<boolean> {
   const provider = new LocalSettingsProvider(projectRoot);
   return await fs.pathExists(provider.localSettingsFilePath);
 }
+
+export async function addSubscriptionAndResourceGroupNode(env: string) {
+  if (!environmentTreeProvider) {
+    return;
+  }
+
+  const parentCommand = environmentTreeProvider.findCommand("fx-extension.environment." + env);
```
Comment (qinezh) [inline packages/vscode-extension/src/envTree.ts:128]: Shall we return directly if the env value equal to local?

Comment (dooriya) [reply packages/vscode-extension/src/envTree.ts:128]: Sounds good as we already include 'local' env node in the tree view now.

**Review by qinezh — APPROVED** [review]

**Review by a1exwang — APPROVED** [review]

**Review by xiaolang124 — APPROVED** [review]

**Review by HuihuiWu-Microsoft — APPROVED** [review]
LGTM

**Pull request merged**